steps/rhelah.py

```diff
@@ -785,3 +786,29 @@ def step_impl(context, container, mount_option, option=None):
 
     assert context.remote_cmd('shell',
                               module_args='grep -E "%s" /proc/mounts' % matches)
+
+@when(u'switch atomic host to "{mode}" mode')
+@when(u'switch atomic host to development mode')
+def step_impl(context, mode="development"):
+    '''Prepare the current deployment for hotfix or development mode'''
+    opt_dict = {"development": "", "hotfix": "--hotfix"}
+    assert context.remote_cmd(cmd='command',
+                              module_args='atomic host unlock %s' % opt_dict[mode])
+
+@then(u'check unlock status "{mode_str}" exists')
+@then(u'check unlock status "{mode_str}" does not exists "{ignore_rc}"')
+def step_impl(context, mode_str, ignore_rc='false'):
+    '''Check unlock status'''
+    rc = string_to_bool(context, ignore_rc)
+    ostree_status = get_ostree_admin_status(context)
```

Comment:
Same note here about checking for a valid value in `ostree_status` before trying to use it